test/validation_test.rb

```diff
@@ -91,11 +91,6 @@ class Measured::Rails::ValidationTest < ActiveSupport::TestCase
     refute thing.valid?
   end
 
-  test "validation fails if only if the unit is set" do
```

Comment:
looks like as you mentioned there is test coverage for this in `test/active_record_test.rb`. we gucci